Orange/widgets/data/owcorrelations.py

```diff
@@ -415,7 +415,7 @@ def commit(self):
 
         # data has been imputed; send original attributes
         self.Outputs.features.send(AttributeList(
-            [self.data.domain[name] for name, _ in self.selection]))
+            [self.data.domain[var.name] for var in self.selection]))
```

Comment:
`self.data.domain[var.name]` is a bit redundant, domain can be indexed with variables as well as names:
`self.data.domain[var]`

Reply:
This should work even if var is loaded from settings and is not the same object right? Should have the same hash even in those exotic cases with compute_values etc? If not sure you can always ignore my comment above and leave it as it is now :)

Reply:
I thought so, too, but it doesn't work. `compute_value` is a part of a hash (and has to be, otherwise domain transformation would mistook to variables to be equivalent when they are not).

We have to use `var.name` here.